cmd/lakefs/cmd/lua.go

```diff
@@ -28,7 +28,7 @@ var luaRunCmd = &cobra.Command{
 
 		ctx := cmd.Context()
 		l := lua.NewStateEx()
-		lualibs.OpenSafe(l, ctx, os.Stdout)
+		lualibs.OpenSafe(l, ctx, lualibs.OpenSafeConfig{NetHTTPEnabled: true}, os.Stdout)
```

Comment:
Do we want to enable it by default?

Reply:
no. the default is disabled. this is for the command line lua runner. in this case I think we want to enable everything. I wasn't sure that in this case I would consider the action configuration.